frame/staking/src/lib.rs

```diff
@@ -830,6 +948,11 @@ decl_module! {
 			}
 		}
 
+		fn submit_election_result(origin) {
+			let who = ensure_signed(origin)?;
+			// TODO: nothing for now, needs encoding.
```

Comment:
So few questions here:
1. How do we plan to deal with multiple results?
2. What about rewards?
3. Should all validators compute phragmen off-chain?
4. We definitely need to validate the results here (or possibly even during transaction validation), is there a fast way of doing that? Isn't that a DoS vector?

Reply:
See the hackmd link above, there is a point about all of your questiuons, which are all : 
1. there is a way to score solutions. TODO
2. I think they should exist, but I should discuss it in the research team. 
3. I assume yes. But let's talk more: what else can be done? 
4. Yeah and this might be a bottleneck.